owncloudData/src/test/java/com/owncloud/android/data/file/datasources/OCLocalFileDataSourceTest.kt

```diff
@@ -36,9 +42,7 @@ class OCLocalFileDataSourceTest {
     private lateinit var dao: FileDao
 
     private val mapper = OCFileMapper()
-
-    private val dummy = OC_FILE.copy(id = 0)
-    private val dummyEntity = mapper.toEntity(OC_FILE)!!
+    private val dummyFileEntity: OCFileEntity = mapper.toEntity(OC_FILE)!!
```

Comment:
I would add this as in a companion and uppercase with snake case `DUMMY_FILE_ENTITY`